### PR TITLE
Fizzbuzz/python/kruize

### PR DESCRIPTION
**Pull request by Tech-Wizz**
I need someone to test this and tell me how the documentation with the website and readme. I got it to work the way Devin wanted us to.

### CODE REVIEW

File: fizzbuzz/python/kruize/FizzBuzz.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    elif (n % 5 == 0):                      #gets just multiple of 5
+        print ('Buzz')
+    elif (n % 3 == 0):                      #gets multiple of 3
+        print ('Fizz')
+    else:                                   #just prints the numbers that don't meet a conditional 
+        print (n)
+
+        
+    n += 1                                  #increases the number each time in the while loop
+
```
Comment (laytonmccafferty) [inline fizzbuzz/python/kruize/FizzBuzz.py:18]: if you want to, you can remove the empty line at the end of this program and have a little less space between the counter and the if statements. a bit nit-picky, but. nice work! +1

**Review by laytonmccafferty — COMMENTED** [review]
just reviewed your code and, like before, it looks nice! +1 I'll pull it down to my machine now and test it

**Comment by Skeegan123** [comment]
When I ran your script on my machine I got the following output:
❯ ./run.sh 20
zsh: permission denied: ./run.sh
❯ source run.sh 15
run.sh:2: command not found: ^M
----------
run.sh:4: command not found: ^M
1
2
Fizz
4
Buzz
Fizz
7
8
Fizz
Buzz
11
Fizz
13
14
FizzBuzz
run.sh:6: command not found: ^M
----------

Finished Running FizzBuzz

For the permission denied error I believe you need to use chmod (See slack), and for the command not found error I am not exactly sure, Odly enough removing the extra lines fixed it for me but I do not know why as my script worked fine with those lines. May be because i use ZSH.

**Comment by laytonmccafferty** [comment]
I'm on Mac as well, so I decided to test out this and see if I got the same issues.

For the permission denied first error, as Keegan said you will need the command from the slack that is `chmod +x run.sh`. However, I am running into an add error after this.
Screen Shot 2023-03-09 at 9 11 30 AM
This error was fixed with the command Keegan mentioned `sed -i -e 's/\r$//' run.sh` but then I ran into a different error about the python command not being found. It works when changing the run.sh file to use python3 instead of python, but I think that may be because I only have python3 installed on my machine. (this can be checked by running `ls usr/bin/python*`)

When everything runs correctly, the output is correct though!
Screen Shot 2023-03-09 at 9 16 25 AM

**Comment by Tech-Wizz** [comment]
I am not sure where the npm command is used. It might be a windows thing but you two still got it to work

**Comment by Skeegan123** [comment]
Alright so I pulled down your latest commit and have a couple more notes. After some more digging, the command not found error is from a weird text thing where using DOS text in a Unix system is causing it. I dont know if there is a way to fix this without breaking it for either your computer or mine/Layton's. You could try the sed command I mentioned above but like I said that might make it not work on your computer. I would maybe suggest adding a note in your documentation suggesting mac users to use the command ```dos2unix run.sh``` if they are having that error (they would first have to install dos2unix to do so). But the final and most important thing is to change the permissions on your run.sh file using the command ```chmod +x run.sh``` then commiting the run.sh file with these permissions. This will allow people to run the script by typing ```./run.sh```. After that you should be completely good to go and the output looks good!

**Comment by Tech-Wizz** [comment]
ok, yah. I will add that to the instructions. Fixing errors I can't see is a little tricky and maybe Devin might have some insight tomorrow on how to tackle this issue.

**Review by Skeegan123 — COMMENTED** [review]
When I said to add it to the instructions I moreso meant the part about the dos2unix but if you want to wait till tomorrow thats fine. Also, i would just run that chmod command on your own computer then commit the file after doing that so that nobody else has to run the command it just works. If you'd rather leave it as is then I'll approve the request but I would suggest asking about it first. Up to you

**Review by Skeegan123 — APPROVED** [review]
+10 After adding the correct permissions and converting from DOS to UNIX line endings, the code works perfectly well.
Screenshot 2023-03-21 at 4 25 50 PM

**Pull request merged**